balancing/balance_breaker.go

```diff
@@ -522,24 +521,28 @@ func (ms *MeasuredStorage) RoundTrip(req *http.Request) (*http.Response, error)
 	log.Debugf("MeasuredStorage %s: Got request id %s\n", ms.Name, reqID)
 	resp, err := ms.RoundTripper.RoundTrip(req)
 	duration := time.Since(start)
-	success := backend.IsSuccessful(resp, err)
+	success := backendSuccess(resp, err)
 	open := ms.Breaker.Record(duration, success)
-	log.Debugf("MeasuredStorage %s: Request %s took %s was successful: %t, opened breaker %t\n", ms.Name, reqID, duration, success, open)
+	log.Debugf("s %s: Request %s took %s was successful: %t, opened breaker %t\n", ms.Name, reqID, duration, success, open)
```

Comment:
You replaced `MeasuredStorage` with `s` in the log message. Is this intentional?